### PR TITLE
1664 gis upload status

### PR DESCRIPTION
**Pull request by marcellmueller**
implements #1664 

Please roast my database code. 

Also fixed the metabase link colour and added a breakpoint for it on the GIS success page since I was there.

### CODE REVIEW

File: db/deploy/mutations/parse_gis_data.sql [hunk 1]
```diff
@@ -22,20 +23,27 @@ begin
...
+        select id into application_row_id from ccbc_public.application where ccbc_number = ccbc_id;
+
+        -- Look up the application_status based on the application_id value
+        select status into application_status_name from ccbc_public.application_status
+        where application_id = application_row_id
+        order by id desc limit 1;
+
+        -- If no row was found, add to list
+        if not found or
+            (application_status_name != 'received' and application_status_name != 'screening' and application_status_name != 'assessment')
```
Comment (AntBush) [inline db/deploy/mutations/parse_gis_data.sql:38]: nit: you can add in a `application_status_name not in ('received', 'screening', 'assessment')` I think. But more of a style choice, might make things easier when we want to add in other statuses

File: db/deploy/computed_columns/gis_data_counts.sql [hunk 2]
```diff
@@ -50,12 +53,32 @@ return query
...
+    not_imported as (
+       select
+            (json_row->>'ccbc_number') as ccbc_number,
+            a.id as application_id,
+            json_row as json_data
+        from json_rows, ccbc_public.application a
+        join ccbc_public.application_status s on s.application_id = a.id
+        where a.ccbc_number = (json_row->>'ccbc_number')
+        and s.archived_at is null
+        and (s.status != 'received' and s.status != 'screening' and s.status != 'assessment')
```
Comment (AntBush) [inline db/deploy/computed_columns/gis_data_counts.sql:75]: nit: can be replaced with same comment done in parse_gis_data.sql

**Review by AntBush — APPROVED** [review]
No roasting here! Looks good, just a couple of nits but otherwise good stuff!

File: db/deploy/mutations/parse_gis_data.sql (hunk 1 above)
Comment (marcellmueller) [reply db/deploy/mutations/parse_gis_data.sql]: Thanks, I definitely think this is simpler. Updated!

**Pull request merged**